crates/oxc_linter/src/rules/eslint/no_return_await.rs

```diff
@@ -31,7 +31,7 @@ declare_oxc_lint!(
     /// }
     /// ```
     NoReturnAwait,
-    correctness
+    pedantic
```

Comment:
Should this be suspicious?

Reply:
Can be both, we can revisit these non-correctness rules later.